astroquery/esasky/tests/test_esasky_remote.py

```diff
@@ -1,352 +1,281 @@
+        file_path = tmp_path / 'ISO-IR'
 
-        all_maps = ESASkyClass.query_object_maps(position="M51")
-        iso_maps = ESASkyClass.query_object_maps(position="M51", missions='ISO-IR')
+        all_maps = ESASky.query_object_maps(position="M51")
+        iso_maps = ESASky.query_object_maps(position="M51", missions='ISO-IR')
         # Remove a few maps, so the other list will have downloadable ones, too
         iso_maps['ISO-IR'].remove_rows([0, 1])
-        ESASkyClass.get_maps(iso_maps, download_dir=download_directory)
+        ESASky.get_maps(iso_maps, download_dir=tmp_path)
         assert len(os.listdir(file_path)) == len(all_maps['ISO-IR']) - 2
 
         iso_maps2 = dict({'ISO-IR': all_maps['ISO-IR'][:2]})
-        ESASkyClass.get_maps(iso_maps2, download_dir=download_directory)
+        ESASky.get_maps(iso_maps2, download_dir=tmp_path)
         assert len(os.listdir(file_path)) == len(all_maps['ISO-IR'])
 
-        shutil.rmtree(download_directory)
-
     def test_esasky_query_region_spectra(self):
-        result = ESASkyClass.query_region_spectra(position="M51", radius="5 arcmin")
+        result = ESASky.query_region_spectra(position="M51", radius="5 arcmin")
         assert isinstance(result, TableList)
 
     def test_esasky_query_object_spectra(self):
-        result = ESASkyClass.query_object_spectra(position="M51")
+        result = ESASky.query_object_spectra(position="M51")
         assert isinstance(result, TableList)
 
     @pytest.mark.bigdata
-    def test_esasky_get_spectra(self):
-        download_directory = "ESASkyRemoteTest"
-        if not os.path.exists(download_directory):
-            os.makedirs(download_directory)
-
-        missions = ESASkyClass.list_spectra()
-        # HST-IR has no data, LAMOST does not support download
-        missions = [mission for mission in missions if mission not in ("HST-IR", "LAMOST")]
-        ESASkyClass.get_spectra(position="M1", missions=missions, download_dir=download_directory)
-
-        for mission in missions:
-            file_path = os.path.join(download_directory, mission)
-            assert file_path[:16] == "ESASkyRemoteTest"
-
-        shutil.rmtree(download_directory)
-
-    def test_esasky_get_spectra_small(self):
-        download_directory = "ESASkyRemoteTest"
-        if not os.path.exists(download_directory):
-            os.makedirs(download_directory)
-
+    @pytest.mark.parametrize("mission", ["XMM-NEWTON", "Chandra", "IUE", "HST-UV",
+                                         "HST-OPTICAL", "ISO-IR", "Herschel"])
+    def test_esasky_get_spectra(self, tmp_path, mission):
+        # Not all missions are tested here:
+        # - HST-IR, JWST_Mid-IR and CHEOPS have no data
+        # - LAMOST does not support download
+        # - JWST_Near-IR returns a zip file with many fits files in it, unsupported
+        ESASky.get_spectra(position="M1", missions=mission, download_dir=tmp_path)
+
+        assert (tmp_path / mission).exists()
```

Comment:
I would rather prefer to use explicit Path syntax than the `/` operator. How about this one instead, here and everywhere below?
```suggestion
        assert Path(tmp_path, mission).exists()
```

Reply:
Fixed.